mlrun/feature_store/feature_set.py

```diff
@@ -1030,6 +1030,56 @@ def deploy_ingestion_service(
             self, source, targets, name, run_config, verbose
         )
 
+    def check_relation(
+        self,
+        relation: Dict[str, Union[str, Entity]],
+        right_fset_fields,
```

Comment:
type hints for all params and function return
relation - default value? Allow type None?